src/main/java/rs/teslaris/teslaris/TeslarisStarter.java

```diff
@@ -0,0 +1,15 @@
+package rs.teslaris.teslaris;
```

Comment:
why teslaris two times???

Reply:
Autogenerated because it joins groupId + artifactId. I can remove the latter.